runtime/vm/jvminit.c

```diff
@@ -2843,8 +2843,41 @@ modifyDllLoadTable(J9JavaVM * vm, J9Pool* loadTable, J9VMInitArgs* j9vm_args)
 					return JNI_ERR;
 				}
 			}
+
```

Comment:
This block of code splits the existing code to calculate the `dllCheckPathPtr` in half.  It would be better to put the new code immediately after the `openFlags = 0` line so the rest of the existing code is kept in a logical block